src/Directory.Packages.props

```diff
@@ -9,6 +9,7 @@
     <PackageVersion Include="AWSSDK.CloudWatch" Version="3.7.301.30" />
     <PackageVersion Include="AWSSDK.SecurityToken" Version="3.7.300.49" />
     <PackageVersion Include="AWSSDK.SQS" Version="3.7.300.49" />
+    <PackageVersion Include="Azure.Core" Version="1.37.0" />
```

Comment:
This reference is not required. Any reasons why we add it?

Reply:
It seemed to be required.